tests/integration_tests/lightgbm_tuner_tests/test_optimize.py

```diff
@@ -949,3 +949,26 @@ def test_optuna_callback(self) -> None:
             tuner._tune_params(["num_leaves"], 10, optuna.samplers.TPESampler(), "num_leaves")
 
         assert callback_mock.call_count == 10
+
+    def test_get_best_booster(self) -> None:
+        unexpected_value = 20  # out of scope.
+
+        params = {"verbose": -1, "lambda_l1": unexpected_value}  # type: Dict
+        dataset = lgb.Dataset(np.zeros((10, 10)))
+
+        study = optuna.create_study()
+        # Test with the argument of `return_cvbooster` is omitted as it requires `lightgbm>=3.0.0`.
```

Comment:
`LightGBM==3.0.0` has been released, and the known issue of  `LightGBMTuner` about `feature_pre_filter` was resolved by https://github.com/optuna/optuna/pull/1774. I think we can test the `return_cvbooster` argument.

Reply:
Sure, noted your points. I will try to make the change tomorrow, thank you so much for your help as always.

Reply:
I think the test for `return_cvbooster` argument is added and the latest commit `ec1dba9` has passed mosts of tests. However only `ci/circleci: tests-python35` failed due to an ImportError for `skorch` below. 


```
==================================== ERRORS ====================================
___________ ERROR collecting tests/integration_tests/test_skorch.py ____________
ImportError while importing test module '/home/circleci/project/tests/integration_tests/test_skorch.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.5/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/integration_tests/test_skorch.py:3: in <module>
    import skorch
E   ImportError: No module named 'skorch'
```

Reply:
Could you merge `master` into this feature branch again? https://github.com/optuna/optuna/pull/1668 introduced the change in `.circleci/config.yml` to ignore the `skorch`-related tests, but it may not be applied to this branch yet.